helix-term/src/ui/prompt.rs

```diff
@@ -430,8 +430,8 @@ impl Component for Prompt {
             }
             alt!('b') | alt!(Left) => self.move_cursor(Movement::BackwardWord(1)),
             alt!('f') | alt!(Right) => self.move_cursor(Movement::ForwardWord(1)),
-            ctrl!('b') | ctrl!(Left) => self.move_cursor(Movement::BackwardChar(1)),
-            ctrl!('f') | ctrl!(Right) => self.move_cursor(Movement::ForwardChar(1)),
+            ctrl!('b') | ctrl!(Left) | key!(Left) => self.move_cursor(Movement::BackwardChar(1)),
```

Comment:
You should replace the `ctrl!()` mapping with `key!()`

Reply:
Done! I was thinking about that, but i didnt do it because it was working before with ctrl too, so i wasnt sure if that was intentional.

Reply:
Yeah looks like it was a bug when transcribing the events: https://github.com/helix-editor/helix/commit/efc2b4c77be55afad07762cdde9b3b74c8477933#diff-910b0deb36d6f016258fafae83ea994d924230f49ab46dce71ff326a4a06e3efL456

Reply:
Ah I see, `key!()` only matches the key with no modifiers, hmm. I think it's okay in this case